services/static-webserver/client/source/class/osparc/workbench/WorkbenchUI.js

```diff
@@ -1939,24 +1939,25 @@ qx.Class.define("osparc.workbench.WorkbenchUI", {
 
       if ("dataTransfer" in e) {
         this.__isDraggingFile = false;
-        const files = e.dataTransfer.files;
-        if (files.length === 1) {
-          const pos = {
-            x: e.offsetX,
-            y: e.offsetY
-          };
-          const fileList = e.dataTransfer.files;
-          if (fileList.length) {
+        const files = osparc.file.FileDrop.getFilesFromEvent(e);
+        if (files.length) {
+          if (files.length === 1) {
+            const pos = {
+              x: e.offsetX,
+              y: e.offsetY
+            };
             const service = qx.data.marshal.Json.createModel(osparc.service.Utils.getFilePicker());
             const nodeUI = await this.__addNode(service, pos);
             if (nodeUI) {
               const filePicker = new osparc.file.FilePicker(nodeUI.getNode(), "workbench");
-              filePicker.uploadPendingFiles(fileList);
+              filePicker.uploadPendingFiles(files);
               filePicker.addListener("fileUploaded", () => this.fireDataEvent("nodeSelected", nodeUI.getNodeId()), this);
             }
+          } else {
+            osparc.FlashMessenger.getInstance().logAs(this.tr("Only one file at a time is accepted."), "ERROR");
           }
         } else {
-          osparc.FlashMessenger.getInstance().logAs(this.tr("Only one file is accepted"), "ERROR");
+          osparc.FlashMessenger.getInstance().logAs(this.tr("Folders are not accepted. You might want to upload a zip file."), "ERROR");
```

Comment:
Maybe a more impersonal message is better. Something like the following.
```suggestion
          osparc.FlashMessenger.getInstance().logAs(this.tr("Folders are not accepted. Upload a zip file."), "ERROR");
```

Reply:
Nah, a like the personal message, also I don't like the imperative in this case.